src/lib/storage/table_key_constraint.cpp

```diff
@@ -2,13 +2,20 @@
 
 namespace opossum {
 
-TableKeyConstraint::TableKeyConstraint(std::unordered_set<ColumnID> init_columns, KeyConstraintType init_key_type)
-    : AbstractTableConstraint(std::move(init_columns)), _key_type(init_key_type) {}
+TableKeyConstraint::TableKeyConstraint(std::unordered_set<ColumnID> init_columns, KeyConstraintType init_key_type,
+                                       std::optional<CommitID> latest_validated_commit /* = std::nullopt*/)
```

Comment:
Is this a debugging artifact? 
```suggestion
                                       std::optional<CommitID> latest_validated_commit)
```

Reply:
I somehow like the idea of knowing default arguments in the .cpp files, maybe this annotation is something we can discuss internally in the fortnightly.

Reply:
Same for me. Find it annoying to look those values up in the header file.